code/parachain/frame/fnft/src/lib.rs

```diff
@@ -57,6 +57,7 @@ pub mod pallet {
 		},
 		PalletId,
 	};
+	use multihash::{Blake2b256, Hasher};
```

Comment:
Is this not supported by a core library already? Let's not introduce new dependencies if they already exists in the core, specially cryptography stuff

Reply:
2b4b66d3fe39fff98c6cb34c748a5710f7ac616f